### PR TITLE
[1.13.x] KOGITO-6090 - detach examples from kogito-build-parent

### PR DESCRIPTION
**Pull request by MarianMacik**
depends on [kiegroup/kogito-runtimes#2060](https://github.com/kiegroup/kogito-runtimes/pull/2060)

A backport of #1052 with some additions to compensate for the changes between branches.

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: dmn-event-driven-quarkus/pom.xml [hunk 1]
```diff
@@ -12,10 +12,13 @@
   <artifactId>dmn-event-driven-quarkus</artifactId>
   <name>Kogito Example :: DMN Event-Driven :: Quarkus</name>
   <properties>
-    <quarkus-plugin.version>2.3.0.Final</quarkus-plugin.version>
+    <quarkus-plugin.version>2.7.2.Final</quarkus-plugin.version>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>2.3.0.Final</quarkus.platform.version>
```
Comment (MarianMacik) [inline dmn-event-driven-quarkus/pom.xml:18]: @evacchi This hasn't been updated, so I updated it. I hope it's OK.

**Comment by kie-ci1** [comment]
**(tests) - kogito-examples job** `#10` was: **UNSTABLE**
Possible explanation: This should be test failures


**Test results:**
- PASSED: 414
- FAILED: 3

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-examples.tests.kogito-examples/10/testReport/org.acme.travel/MultiMessagingIT/Build_projects___testProcess/">Build projects / org.acme.travel.MultiMessagingIT.testProcess</a></summary>
org.opentest4j.AssertionFailedError: expected: <0> but was: <3><br/>	at org.acme.travel.MultiMessagingIT.testProcess(MultiMessagingIT.java:104)<br/>
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-examples.tests.kogito-examples/10/testReport/org.acme.travel/MessagingIT/Build_projects___testProcess/">Build projects / org.acme.travel.MessagingIT.testProcess</a></summary>
org.opentest4j.AssertionFailedError: expected: <0> but was: <3><br/>	at org.acme.travel.MessagingIT.testProcess(MessagingIT.java:109)<br/>
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-examples.tests.kogito-examples/10/testReport/org.kie.kogito.examples.demo/OutboxIT/testSendProcessEvents/">org.kie.kogito.examples.demo.OutboxIT.testSendProcessEvents</a></summary>
org.awaitility.core.ConditionTimeoutException: Condition with lambda expression in org.kie.kogito.examples.demo.OutboxIT was not fulfilled within 4 minutes  10 seconds.<br/>	at org.kie.kogito.examples.demo.OutboxIT.testSendProcessEvents(OutboxIT.java:70)<br/>Caused by: java.net.ConnectException: Connection refused (Connection refused)<br/>	at org.kie.kogito.examples.demo.OutboxIT.lambda$testSendProcessEvents$0(OutboxIT.java:74)<br/>
</details>

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-examples.tests.kogito-examples/10/display/redirect

**Review by cristianonicolai — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by kie-ci1** [comment]
**(tests) - kogito-examples job** `#11` was: **UNSTABLE**
Possible explanation: This should be test failures


**Test results:**
- PASSED: 416
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-examples.tests.kogito-examples/11/testReport/org.acme.travel/MessagingIT/Build_projects___testProcess/">Build projects / org.acme.travel.MessagingIT.testProcess</a></summary>
org.opentest4j.AssertionFailedError: expected: <0> but was: <1><br/>	at org.acme.travel.MessagingIT.testProcess(MessagingIT.java:109)<br/>
</details>

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-examples.tests.kogito-examples/11/display/redirect

**Comment by MarianMacik** [comment]
Jenkins retest this

**Review by ricardozanini — APPROVED** [review]
Great work!

**Review by antncl — APPROVED** [review]

**Pull request merged**